kinode/src/net/types.rs

```diff
@@ -103,13 +103,15 @@ impl Peers {
                 .key()
                 .clone();
             self.peers.remove(&oldest);
```

Comment:
doesn't need to be in this PR, but we should write an issue or a TODO to ourselves to make net & vfs cull in a more similar way (unless there is a specific reason to have them work differently)

Reply:
my thinking:

opening a new file has <1ms overhead, therefore, culling should happen rarely so as not to 2x or 3x overhead

connecting to a new peer has >10ms overhead due to network latency, therefore, culling old peer can happen every time without adding measurable overhead

Reply:
seems plausible, thanks for explaining